README.md

```diff
@@ -368,6 +368,13 @@ Please see the 'Commit comment' section for more details.
 If it is set to `true`, this action automatically pushes the generated commit to GitHub Pages branch.
 Otherwise, you need to push it by your own. Please read 'Commit comment' section above for more details.
 
+#### `annotate-always` (Optional)
+
+- Type: Boolean
+- Default: `false`
+
+If it is set to `true`, this action will leave check annotations comparing the current benchmark with previous.
```

Comment:
Would you add description which explains what is annotation? I think user cannot know what it is only from this documentation.